api.go

```diff
@@ -167,8 +169,20 @@ type (
 		CreatedAt     int64  `json:"createdAt"`
 		InputAssetID  string `json:"inputAssetId,omitempty"`
 		OutputAssetID string `json:"outputAssetId,omitempty"`
-		Type          string `json:"type"`
-		Params        struct {
+		Output        struct {
+			Export struct {
+				IPFS struct {
```

Comment:
Maybe make these pointers since they won't be present in every response. Otherwise, there would be no simple way to check whether something was there or not on the raw respponse.
```suggestion
		Output        *struct {
			Export *struct {
				IPFS *struct {
```